feedback_prompt_learning/search_algo/mcts.py

```
@@ -0,0 +1,931 @@
+        return batch
+    
+    async def run(self) -> MCTSNode:
+        """Main MCTS algorithm"""
+        self.logger.info(f"\nStarting MCTS with {self.num_iterations} iterations...")
+        self.logger.info(f"Initial prompt: {self.initial_prompt}\n")
+        
+        # Initialize root - evaluate on eval batch for unbiased reward
+
+        # Root is at depth 0, get eval batch for it
+        eval_batch = self._get_eval_batch_for_depth(0)
+        self.root.reward, eval_evaluations = await self._evaluate_prompt(self.root.prompt, eval_batch)
```

Ruff (0.14.7)

Unpacked variable `eval_evaluations` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)